build.gradle

```diff
@@ -206,33 +215,22 @@ task yarn(dependsOn: setupYarn) {
 }
 
 task buildEnigma(type: Zip) {
+	group = buildMappingGroup
 	from mappingsDir
 	include "**/*"
 	archiveName "yarn-enigma-${yarnVersion}.zip"
 	destinationDir(file("build/libs"))
 }
 
-task checkMappings {
```

Comment:
Add this back for now, it can be looked at more in another PR.

Reply:
Fixed.